modules.py

```diff
@@ -261,42 +223,44 @@ def default_values(var_category_names):
     wave_dampings = ([0, 0.13, 0.17], '[-]')            
 
     if any('x_wec' in i for i in var_category_names):
-        vals['capture_width'] = (4, '[m]')     #12
+        vals['capture_width'] = (76, '[m]')
 
     if any('x_type_wec' in i for i in var_category_names):
         vals['wec_type'] = ('point absorber', '[-]')
         
     if any('x_pen' in i for i in var_category_names):
-        vals['pen_diameter'] = (15, '[m]')    #20 
-        vals['pen_height'] = (4, '[m]')      #6  
-        vals['spacing'] = (150, '[m]')          
-        vals['stock_density'] = (20 , '[kg/m^3]') #20
-        vals['pen_depth'] = (10, '[m]')         
+        vals['pen_diameter'] = (25, '[m]') 
+        vals['pen_height'] = (10, '[m]')  
+        vals['stock_density'] = (30 , '[kg/m^3]')
    
     if any('p_pen' in i for i in var_category_names):
-        vals['num_pens'] = (18, '[-]')          
+        vals['num_pens'] = (18, '[-]')  
+        vals['spacing'] = (150, '[m]') 
+        vals['pen_depth'] = (10, '[m]')   
+        vals['pen_unit_cost'] = (100, '[$/m^3]')    # 80 $/m^3 for net pen + 20 $/m^3 for mooring
+        vals['permeability'] = (0.8, '[-]')      
         
     if any('x_env' in i for i in var_category_names):
-        vals['temp'] = (16, 'C')               
-        vals['salinity'] = (33, '[PSU]')
-        vals['U'] = (.2, '[m/s]')
-        vals['O2_in'] = (8,'[mg/l]')                      
-        vals['wave_height'] = (1.4, '[m]')     
-        vals['wave_period'] = (8.33, '[s]')     
+        vals['temp'] = (10.07, 'C')
+        vals['O2_in'] = (9.44,'[mg/l]')
+        vals['wave_height'] = (1.04, '[m]')
+        vals['wave_period'] = (5.73, '[s]') 
+    
+    if any('p_env' in i for i in var_category_names):
+        vals['salinity'] = (31.74, '[PSU]')
+        vals['U'] = (.1, '[m/s]')
     
     if any('p_wec' in i for i in var_category_names):
         vals['wec_unit_cost'] = (0.45 * 1.19, '[$/kWh]')   # 'point absorber' * inflation rate from 2014 to 2022
-        vals['pen_unit_cost'] = (100, '[$/m^3]')    # 80 $/m^3 for net pen + 20 $/m^3 for mooring
-        vals['permeability'] = (0.8, '[-]')
         vals['capture_width_ratio_dict'] = (dict(zip(wec_types[0], capture_width_ratios[0])), '[-]')
         vals['wave_damping_dict'] = (dict(zip(wec_types[0], wave_dampings[0])), '[-]')
         
     if any('p_fish_salmon' in i for i in var_category_names):
         vals['F_p'] = (0.45, '[-]')
```

Comment:
Consider relisting the definitions for each of these variables so that you don't have to scroll back and forth if you forget. 
```suggestion
        vals['F_p'] = (0.45, '[-]')
```

Reply:
updated.